spec/integration/aws_server_certificate_spec.rb

```diff
@@ -4,97 +4,79 @@
   extend AWSSupport
 
   # http://openssl.6102.n7.nabble.com/create-certificate-chain-td44046.html
+  # Follow those instructions except the chain should be `ca-int.crt` only
+  # instead of concatenated intermediate and root
 
   when_the_chef_12_server "exists", organization: 'foo', server_scope: :context do
     with_aws "without a VPC" do
       cert_string = <<-CERT
 -----BEGIN CERTIFICATE-----
-MIICvjCCAaYCAkE4MA0GCSqGSIb3DQEBDQUAMBgxFjAUBgNVBAMTDVppbnRlcm1l
-ZGlhdGUwHhcNMTcwMzE0MTQ0NzQzWhcNMjcwMzEyMTQ0NzQzWjAxMQ4wDAYDVQQD
-EwVhbGljZTEfMB0GCSqGSIb3DQEJARYQYWxpY2VAcmFuZG9tLmNvbTCCASIwDQYJ
-KoZIhvcNAQEBBQADggEPADCCAQoCggEBANQep8AvCZ4pLTDahkoQlvj+R8heF74e
-vH34vfyU5TNrmoWTvlGDEwoCVEEcI8M4R2g9XYhB7SIqTDbJn5T6aLTsaTyMr/g0
-p6Fiofd/k5pX5XuJDvya3e91Ixbu/qvvHyNlnLU1ba7HotqXcTJqMiW0rYEVE+PV
-KctGByOlp5d34ytFguZZsPVR+Ma7HKb55ksIOkEa7VNQhL6hGRU+Ivnuvu/TJtmZ
-lzb/9GUrs5L6JH2dMW9LUzvIT5rbAM2ntm4R1W+tU7Vb8gX6Q1ir5oTfRzENAqJZ
-uFBGUaLhJzJhufD40c25cvgUVFQaRghFjHdqf2pqQsAS8LJYbj0uR8MCAwEAATAN
-BgkqhkiG9w0BAQ0FAAOCAQEAEvmMK6C0AU2ww5t+/mS65laGTKdlVBvm3qqX1Fo1
-MDBCVqVc51m69lwHgNxWzt8dcEFFT/d1xlDE2JWleSqH5/Dsj9AcYla4hTr5f76c
-7M4LcYwuO+s/eTFkTl92DJiXVDczlsN/w1+lvt3Uw5ReKlMBMK+dbPUpZ/yl+NrU
-3pf98aoP7m+oocl5lxksHse+klOkvlT26GfQ7JgGlWIO2xAhUtNkh6feFxZ1dP0O
-45RTqOorOuxSV6PbgEahmsElRbAmxs/iSRi22340R7/rd8+g7Y+3olBFJsrzcYr4
-1t8FiEUNUxoJIIPnHi/QIQxzmiQ3LZNEmtCnAPMeci1gug==
------END CERTIFICATE-----
+MIICyjCCAbICAnyXMA0GCSqGSIb3DQEBDQUAMCcxJTAjBgNVBAMTHENoZWZQcm92
+aXNpb25pbmdJbnRlcm1lZGlhdGUwHhcNMTcwODI0MTY0NTQyWhcNMjIwODIzMTY0
+NTQyWjAuMQ4wDAYDVQQDEwVhbGljZTEcMBoGCSqGSIb3DQEJARYNYWxpY2VAY2hl
+Zi5pbzCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBAKxeXpigv/i4OWPB
+kIBV3+HrKnEh05uEaq4UfJw0p6opKs4hyc92SvcFge4YBcLRbzhyMY5fUZAJXEla
+csb6lEs2DMlW/KZGvfSMts2tVNbFVSsIsuSfhHVr9kemE42RPrtsO/0chOk2P/dl
+P/KvXRF9AtEQe27/CWnJywCkP6tT6baZM6X+GGgAPUHvxN4BmJzz6uHpMVH+rBbb
+t9ruLoSdX0zbaTRLesBC5Hc8uK2wzvDx0pUj+aKcWg5mtPBT6yReH6D5ePV2Jf10
+9FGKMqPN6tOO6ZyAIWuKx3v09JzxmWGxNEyR65SNiI+ft092UFEKXYfgK58HZlWj
+pBcOsHECAwEAATANBgkqhkiG9w0BAQ0FAAOCAQEAY1KXZv35hUER0WZz7JMKlvhI
+IUpfB2NibP9G5LhtxFY1fa+MLp9mJ+yI3hg0x+6xBPTiPDGVpFdJ+LH/mKWxsaWT
+17ZRGOeG2gZAlGr8Y8IMuR5cv2fmG6zCObMFxFR89pKvAYR+H5RFcHBg/N86SX6D
+/KKNXu+NC9z/lrNVU6J/XUCk54YdmjMvadHs1aJp6NWH7LI9df27AlBGLnRc/w04
+agCh4aCsjczD2YeBVl6idws/InYbSwhz7x9zXz2qB3BbI3psgBfJQQcJukulVGza
+RkD993U+CJpJMDbpIGRe9lXe33R3tGbbvfa4FaXZlZgoKCrE21SCb2hxlNG2+w==
+-----END CERTIFICATE-----           
```

Comment:
extra spaces are been added after -----END CERTIFICATE-----

Reply:
It's part of server certificate so can't remove it